spec/models/ext_management_system_spec.rb

```diff
@@ -85,6 +85,15 @@
     expect(described_class.ems_infra_discovery_types).to match_array(expected_types)
   end
 
+  it ".with_eligible_manager_types" do
+    v = FactoryBot.create(:ems_vmware)
+    r = FactoryBot.create(:ems_redhat)
+
+    expect(described_class.with_eligible_manager_types([v.class, r.class]).count).to eq(2)
+    expect(described_class.with_eligible_manager_types([v.class]).count).to eq(1)
+    expect(described_class.with_eligible_manager_types([r.class]).count).to eq(1)
```

Comment:
Should one of these expectations be `expect(described_class.with_eligible_manager_types(r.class).count).to eq(1)` without the `[]`?

Reply:
Good call.